viz-lib/src/visualizations/chart/plotly/index.ts

```diff
@@ -8,7 +8,6 @@ import updateAxes from "./updateAxes";
 import updateChartSize from "./updateChartSize";
 import { prepareCustomChartData, createCustomChartRenderer } from "./customChartUtils";
 
-// @ts-expect-error ts-migrate(2339) FIXME: Property 'setPlotConfig' does not exist on type 't... Remove this comment to see the full error message
```

Comment:
Is this change intentional?

Reply:
@lucydodo 

Thank for seeing the PR !

> Is this change intentional?

Yes.
As the latest `@types/plotly.js` have setPlotConfig type definition , we need to remove the line to avoid the error.